src/components/editor/utils.ts

```diff
@@ -1 +1,15 @@
 export const isMac = navigator.platform.toLowerCase().includes('mac')
+
+export const removeHTMLTag = (input: string): string => {
+  return input.replace(/(?:<(?:[^>]+)>)/gi, '')
+}
+
+export const removeMarkdownSyntax = (input: string): string => {
+  return input
+    .replace(/\*(.*?)\*/gi, '$1') // bold & italic
+    .replace(/\+\+(.*?)\+\+/gi, '$1') // underline
+    .replace(/~(.*?)~/gi, '$1') // subscript & strikethrough
+    .replace(/\^(.*?)\^/gi, '$1') // superscript
+    .replace(/`(.*?)`/gi, '$1') // code
+    .replace(/!?\[(.*?)]\(.*?\)/gi, '$1') // link & image
+}
```

Comment:
You're regex are not working 100% perfect.
e.g. if you
- input `1*2*3 4*5**` then the result should be `123 4*5**` but it is `123 45*`
- input `**1 2*` then the result should be `*1 2` but it is `1 2*`

As much as I appreciate that you try to keep this simple and independent from other components, I think a better solution would  to use the DOM results, because then we have the output from the actual markdown renderer.

Reply:
(also you forgot `_asd_`)

Reply:
fixed by using the render output